common/lib/xmodule/xmodule/modulestore/tests/factories.py

```diff
@@ -611,7 +612,8 @@ def mongo_uses_error_check(store):
     Does mongo use the error check as a separate message?
     """
     if hasattr(store, 'mongo_wire_version'):
-        return store.mongo_wire_version() <= 1
+        return store.mongo_wire_version()() <= 1 if isinstance(store.mongo_wire_version(), collections.Callable) \
```

Comment:
Why do we need to call it twice sometimes?  Is this somethnig that should be fixed in the function in the store class?